src/devtools/views/Components/TreeContext.js

```diff
@@ -157,8 +177,12 @@ function reduceTreeState(store: Store, state: State, action: Action): State {
             ((selectedElementIndex: any): number)
           );
           if (selectedElement !== null && selectedElement.parentID !== null) {
-            selectedElementIndex =
-              store.getIndexOfElementID(selectedElement.parentID) || 0;
+            const parentIndex = store.getIndexOfElementID(
+              selectedElement.parentID
+            );
+            if (parentIndex !== null) {
+              selectedElementIndex = parentIndex;
```

Comment:
nit: I guess we could avoid the unnecessary index-to-element lookup below here too?

```js
if (parentIndex !== null) {
  lookupIDForIndex = false;

  selectedElementID = selectedElement.parentID;
  selectedElementIndex = parentIndex;
}
```

Dunno that it's necessarily an important change.